### PR TITLE
Release 0.5.0

### PR DESCRIPTION
**Pull request by D4nte**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,7 +33,7 @@ install_tomlfmt: install_rust
 ## User install
 
 install:
-	$(CARGO) install --force --path .
+	$(CARGO) install --force --path . $(BUILD_ARGS)
```
Comment (thomaseizinger) [inline Makefile]: I disagree with this. `install` is not the `build` command and neither are `clippy`, `test` and `doc`. If we want/need to customize those, I propose to add more variables.

`install` for example does a release build by default, so there is no need to pass this.

Comment (D4nte) [reply Makefile]: This is not for the "release" part, it's for the feature flag!

Comment (thomaseizinger) [reply Makefile]: True, we should rename it to FEATURES then or something like that :)

Comment (thomaseizinger) [reply Makefile]: I'll incorporate that into my open PR!

Comment (thomaseizinger) [reply Makefile]: Actually, if we merge my open PR, you can just rebase this one and don't have to do anything else, I think :)

**Pull request merged**